actuators/fan_controller.py

```diff
@@ -0,0 +1,128 @@
+# Turns on fan if too hot
+
+import argparse
+import json
+import time
+
+import redis
+import glog
+import board
+import digitalio
+
+def parse_args():
+    """ Parse command line args to get temperature and gpio pin """
+    parser = argparse.ArgumentParser(prog="kates programe")
```

Comment:
prog is still kates programe pls do the needful

Reply:
no